front/src/modules/simulationResult/components/SpeedSpaceChart/d3Helpers.ts

```diff
@@ -283,17 +283,18 @@ export const drawSpeedSpaceTrain = (
       CHART_AXES.SPACE_SPEED,
       'speed'
     );
-    if (dataSimulation.standardMarginSpeedData) {
-      drawCurve(
-        chartLocal,
-        'speed margins',
-        dataSimulation.standardMarginSpeedData,
-        'speedSpaceChart',
-        'curveLinear',
-        CHART_AXES.SPACE_SPEED,
-        'margins_speed'
-      );
-    }
+    // TODO: Uncomment this part to show the provisional curve once the GEV v2 is implemented
+    // if (dataSimulation.standardMarginSpeedData) {
+    //   drawCurve(
+    //     chartLocal,
+    //     'speed margins',
+    //     dataSimulation.standardMarginSpeedData,
+    //     'speedSpaceChart',
+    //     'curveLinear',
+    //     CHART_AXES.SPACE_SPEED,
+    //     'margins_speed'
+    //   );
+    // }
```

Comment:
It's a bad practice to leave a deadcode like this. Ideally, it should be removed. If we need to retrieve what was done previously we can always use git :wink: